### PR TITLE
Spring Boot and R2DBC

### PR DESCRIPTION
**Pull request by Michaelin007**

### CODE REVIEW

File: spring-reactive-modules/spring-reactive-data/src/main/java/com/baeldung/r2dbc/configuration/R2DBCConfiguration.java [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import io.r2dbc.h2.H2ConnectionConfiguration;
 import io.r2dbc.h2.H2ConnectionFactory;
+import io.r2dbc.spi.Connection;
```
Comment (Maiklins) [inline spring-reactive-modules/spring-reactive-data/src/main/java/com/baeldung/r2dbc/configuration/R2DBCConfiguration.java:5]: Do we need these imports?

File: spring-reactive-modules/spring-reactive-data/src/test/java/com/baeldung/r2dbc/R2dbcApplicationRdbcTemplateIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.r2dbc.core.R2dbcEntityTemplate;
+import org.springframework.data.relational.core.query.Query;
+import org.springframework.r2dbc.core.DatabaseClient;
+import org.springframework.test.context.junit4.SpringRunner;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Hooks;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
```
Comment (Maiklins) [inline spring-reactive-modules/spring-reactive-data/src/test/java/com/baeldung/r2dbc/R2dbcApplicationRdbcTemplateIntegrationTest.java:21]: The three imports for `Query`, `Flux`, and `Mono` aren't used, so let's remove them.

File: spring-reactive-modules/spring-reactive-data/src/main/java/com/baeldung/r2dbc/configuration/R2DBCConfiguration.java (hunk 1 above)
Comment (Michaelin007) [reply spring-reactive-modules/spring-reactive-data/src/main/java/com/baeldung/r2dbc/configuration/R2DBCConfiguration.java:5]: Thanks @Maiklins, we don't need it. Removed

File: spring-reactive-modules/spring-reactive-data/src/test/java/com/baeldung/r2dbc/R2dbcApplicationRdbcTemplateIntegrationTest.java (hunk 2 above)
Comment (Michaelin007) [reply spring-reactive-modules/spring-reactive-data/src/test/java/com/baeldung/r2dbc/R2dbcApplicationRdbcTemplateIntegrationTest.java:15]: All unused imports were removed. Thanks

**Comment by asif-anwar** [comment]
retest this please

**Comment by Michaelin007** [comment]
> retest this please

Hi @asif-anwar, done.

**Pull request merged**